Source/WebKit/WebProcess/Plugins/PDF/PDFPluginAnnotation.h

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(LEGACY_PDFKIT_PLUGIN)
 
+#include "PDFPluginBase.h"
```

Comment:
This include shouldn't be necessary! Do you mind removing it on your next rebase?

Reply:
Looks like it! Fixed